tests/protocol/polyvox/test.js

```diff
@@ -14,7 +14,6 @@ nitpick.perform("Polyvox protocol sanity - TEST REQUIRES SERVER", Script.resolve
     entityProperties.dimensions = { x: 2000.0, y: 2000.0, z: 2000.0 };
     entityProperties.voxelVolumeSize = { x:23, y: 29, z: 31 };
     entityProperties.voxelData="ads23SGResLiIUYoh987fTREdgdgDGdgdgDSDLLIJklj65edrcgddQ7Y"
-    entityProperties.PolyVoxSurfaceStyle = 3;
```

Comment:
We aren't testing voxelSurfaceStyle now.

Reply:
I know.... When I set this, I get a null (undefined) value on reading the value back.  Do you have an idea why?

Reply:
"voxelSurfaceStyle" not "PolyVoxSurfaceStyle"

Reply:
Ah ... the code comments are very confusing:
Line 1066 in EntityItemProperties:
` * @property {Entities.PolyVoxSurfaceStyle} voxelSurfaceStyle=2 - The style of rendering the voxels' surface and how `
Line 82 in PolyVoxEntityItem:
`     * @typedef {number} Entities.PolyVoxSurfaceStyle`

Now I see that is the type, not the name......

Fixed.